charts/rstudio-pm/NEWS.md

```diff
@@ -1,9 +1,17 @@
 # Changelog
+## 0.5.32
+
+- Bump Chronicle Agent to version 2024.09.0
 
 ## 0.5.31
 
 - Add documentation about PostgreSQL database configuration and mounting passwords from secrets as an env variable
 
+
+## 0.5.31
+
+- Bump Chronicle Agent to version 2024.09.0
+
```

Comment:
Should this still be here? Or is this now covered by the same note in `0.5.32`?

Reply:
We should delete this